macro/Cargo.toml

```diff
@@ -15,3 +15,4 @@ proc-macro = true
 [dependencies]
 proc-macro-hack = { version = "0.5.13" }
 getrandom = "0.2"
+sha2 = { version = "0.9", optional = true, default-features = false }
```

Comment:
I realize I am totally contradicting myself, but looking at the transitive dependencies, maybe we can find something a little lighter?

I found https://crates.io/crates/tiny-keccak which does sha-3. Is there anything better than that?